README.md

```diff
@@ -20,16 +21,16 @@ In the meantime you can install the latest release of the package directly from
 if (!requireNamespace("remotes", quietly = TRUE)) {
   install.packages("remotes")
 }
-remotes::install_github("Roche-GSK/admiral.test") # This is a required dependency of {admiral}
-remotes::install_github("Roche-GSK/admiral")
+remotes::install_github("pharmaverse/admiral.test") # This is a required dependency of {admiral}
```

Comment:
I will change the default branch of {admiral.test} to `main` as well in order to be consistent. Thus, please add `ref = "main"` here as well.